lib/setup.js

```diff
@@ -36,6 +36,11 @@ var constructResponse = function (server) {
           path: route.path,
           plugin: route.public.realm.plugin || null
         };
+      }).sort(function (a, b) {
```

Comment:
The problem is the result of `localeCompare` will never be 0 on the paths; you can't have overlapping paths with hapi. That's why I added the method in with the path.

Agreed about the return type though and the `localeCompare`.